core/db/curibio/alembic/versions/04bfacd4385d_pulse_0_32_1.py

```diff
@@ -11,7 +11,7 @@
 
 # revision identifiers, used by Alembic.
 revision = '04bfacd4385d'
-down_revision = 'ac1e440896f9'
+down_revision = '0b1999fb5ce5'
```

Comment:
How come we're adding 0.32.2 here before it's been published?

Reply:
I ran alembic heads and saw that there are two heads so I switched to prevent alembic branching. Now when I run alembic head I get the new revision as the head instead of two

Reply:
Can this be changed on line 4 as well?